model-client/src/commonMain/kotlin/org/modelix/model/client2/ModelClientV2.kt

```diff
@@ -137,7 +137,7 @@ class ModelClientV2(
                 takeFrom(baseUrl)
                 appendPathSegments("repositories")
             }
-        }.bodyAsText().lines().map { RepositoryId(it) }
+        }.bodyAsText().lines().filter { it.isNotEmpty() }.map { RepositoryId(it) }
```

Comment:
Why can empty lines happen?

Reply:
I explained it in the commit message.

Reply:
So the response includes a single newline and lines emits one empty line in that case?

Reply:
I don't remember exactly anymore but I think the response body was empty. So `lines` returns the empty string.